tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb

```diff
@@ -74,7 +74,7 @@
 
     # Expected exit_code is 0 since this is a vegas shell cmd.
     # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
```

Comment:
Hi Mike,

Thanks for the comment. It looks like that the current beaker test script for name server already uses the puppet resource. It also uses vsh to do the additional check for confirmation, I guess.

Thanks,
Edward

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-puppet-module <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, December 9, 2015 at 6:20 AM
To: cisco/cisco-network-puppet-module <cisco-network-puppet-module@noreply.github.com<mailto:cisco-network-puppet-module@noreply.github.com>>
Cc: edpham <edpham@cisco.com<mailto:edpham@cisco.com>>
Subject: Re: [cisco-network-puppet-module] Fixing internal defect for name_server beaker test (#104)


In tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb<https://github.com/cisco/cisco-network-puppet-module/pull/104#discussion_r47095181>:

> @@ -74,7 +74,7 @@
>
>      # Expected exit_code is 0 since this is a vegas shell cmd.
>      # Flag is set to true to check for absence of RegExp pattern in stdout.
> -    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')


@edwardpham<https://github.com/edwardpham> I should have mentioned this yesterday but we can actually remove these steps all together since they are vsh related. We are still using the puppet resource command to verify functionality.

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-puppet-module/pull/104/files#r47095181>.